### DISCUSSION TITLE
Can I run this on windows?

### DISCUSSION THREAD

**Discussion by nexon33** [coder/code-server#3554] [Q&A] [answered]
Can I run this on windows? The description says run on any machine, but there is no installation guide for windows. I could of course use wsl to run it on windows but I wonder if I could just set it up natively instead of wsl.

**Comment by vital987** [comment, marked as answer]
* Enable Windows WSL
* Install Ubuntu 20.04(current lts release) from Windows Store
* Install Docker from [here](https://desktop.docker.com/win/stable/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=dd-smartbutton&utm_location=header) 
* Get the docker container from [here](https://hub.docker.com/r/linuxserver/code-server), also you can checkout my [project](https://hub.docker.com/r/vital987/code-server) , it's pre-installed with all the dependancies
* Open Docker App > go to Settings > Resources > WSL Integration > Enable it for your distro(ubuntu in this case).
* Enable Docker server
* Setup Ubuntu distro, pull any of the 2 docker images
* Mount your disk volume to store files locally (in wsl, drives are mounted in /mnt/\<drive-letter\>) else you can download the files too, my project also has google drive saving feature so try it if you need it.
* Done!

**Comment by nexon33** [comment]
Just install windows subsystem for linux, then follow the usual install
steps on like ubuntu or something

Op ma 14 jun. 2021 19:41 schreef Atlas ***@***.***>:
